DynamicTablesPkg/Include/ArmNameSpaceObjects.h

```diff
@@ -1053,6 +1053,11 @@ typedef struct CmArmPciInterruptMapInfo {
   */
   UINT8                       PciInterrupt;
 
+  // MU_CHANGE [BEGIN] Allow user to specify Legacy INT object 
+  /// Pci Int Link Name
+  CHAR8   *PciIntLinkName;

```

Comment:
Is there a standard definition like some other fields in this structure? If so, can you please add the section number of the spec/doc for reference? Otherwise, could you please add some more information for this field in the comment? (seems to be 4 Characters, according to your issue description?)